src/lib.rs

```diff
@@ -239,6 +239,9 @@ mod util;
 mod redactable;
 pub use redactable::Redactable;
 
+mod builder;
+pub use builder::{RedactWrapped, Redactor, RedactorBuilder};
```

Comment:
should all of those be exposed on the top level? I feel like RedactorBuilder and RedactWrapped aren't types that need to be named often, and could be shoved into a submodule instead